### PR TITLE
Add Simulation run to CI build pipeline

### PR DESCRIPTION
**Pull request by azvanderpas**

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -157,7 +157,7 @@ public final Boolean isEnabled() {
     public static final double goalHeight = 8.8; //Height of the goal in ft from the carpet
     
     // Shooter Constants
-    public static final int shooterID = 4; // ID of the shooter
+    public static final int shooterID = 14; // ID of the shooter
```
Comment (azvanderpas) [inline src/main/java/frc/robot/Constants.java]: Conflicts with drivebase motor ids (kitbot).

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -32,3 +31,11 @@ jobs:
     # Runs a single command using the runners shell
     - name: Compile and run tests on robot code
       run: ./gradlew build
+
+    # Runs a simulation to spot check crashes
+    - name: Run simulation to check for runtime errors
+      run: timeout 1m ./gradlew simulateJava 2>&1 | awk -v rc=0 '/The robot program quit unexpectedly/ { rc=1 } 1; END {exit rc}'
```
Comment (caseyjbrotherton) [inline .github/workflows/main.yml:37]: If I run the command above without Robot errors, then the return code is 143. ( SIGTERM sent )

seems that awk is receiving the sigterm from timeout?

**Review by caseyjbrotherton — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 2 above)
Comment (azvanderpas) [reply .github/workflows/main.yml:37]: It seems to work in GitHub's linux environment (CI pipeline is clean). Where are you running the command?

Comment (caseyjbrotherton) [reply .github/workflows/main.yml:37]: Running Linux extensions for windows.  There might be subtle difference in signaling.  will go ahead and close this if it works in github's linux

**Review by caseyjbrotherton — APPROVED** [review]

File: src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (sarango2158) [reply src/main/java/frc/robot/Constants.java]: On MDF bot:
driveRightFront = 7
driveRightRear = 8
driveLeftFront = 1
driveLeftRear = 2
intakeMotorOneID = 3
CDSBeltID = 6
intakeWheelOneID = 12
intakeWheelTwoID = 11
shooterCargoID = 5
shooterID = 4

**Review by sarango2158 — APPROVED** [review]

**Pull request merged**